providers/go-feature-flag/pom.xml

```diff
@@ -56,12 +56,6 @@
             <scope>test</scope>
         </dependency>
```

Comment:
updating the readme according to the new design can be helpful, including some explanation about the wasm usage, how it works, where is the wasm evaluation source repository etc'.